test/async-hooks/test-httpparser.response.js

```diff
@@ -19,11 +19,8 @@ const hooks = initHooks();
 hooks.enable();
 
 const request = Buffer.from(
-  'HTTP/1.1 200 OK' + CRLF +
-  'Content-types: text/plain' + CRLF +
-  'Content-Length: 4' + CRLF +
-  CRLF +
-  'pong'
+  `HTTP/1.1 200 OK${CRLF}Content-types: text/plain${CRLF}` +
+  `Content-Length: 4${CRLF}${CRLF}pong`
```

Comment:
I think the thing to do is to go back to the old code layout, but replace lines 22, 23, and 24 with template literals:

```js
const request = Buffer.from(
  `HTTP/1.1 200 OK${CRLF}` +
  `Content-types: text/plain${CRLF}` +
  `Content-Length: 4${CRLF}` +
  CRLF +
  'pong'
);
```

@erdun Does that seem good to you? Can you make that change?